src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -175,7 +175,7 @@ private bool CreateSaveCdkDirectory(string saveCdkDirectoryPath)
         {
             var errorMessage = string.Empty;
             var isValid = true;
-            var targetApplicationDirectoryFullPath = _directoryManager.GetDirectoryInfo(_targetApplicationFullPath).Parent.FullName;
+            var targetApplicationDirectoryFullPath = _directoryManager.GetDirectoryInfo(_targetApplicationFullPath).Parent!.FullName;
```

Comment:
I'd prefer if we do some null handling here instead of suppressing it.

Reply:
Done

Reply:
Done